arcgis-ios-sdk-samples/Scenes/Orbit camera around object/README.md

```diff
@@ -0,0 +1,46 @@
+# Orbit camera around object
+
+Fix the camera to point at and rotate around a target object.
+
+![Image of orbit camera around object](orbit-camera-around-object.png)
+
+## Use case
+
+The orbit geoelement camera controller provides control over the following camera behaviors:
+
+* automatically track the target
```

Comment:
I feel it is better to start all this bullet points in upper case instead of lower case